components/org.wso2.carbon.identity.api.server.userstore/org.wso2.carbon.identity.api.server.userstore.v1/src/gen/java/org/wso2/carbon/identity/api/server/userstore/v1/UserstoresApi.java

```diff
@@ -278,4 +279,27 @@ public Response updateUserStore(@ApiParam(value = "Current domain id of the user
         return delegate.updateUserStore(userstoreDomainId,  userStoreReq );
     }
 
+    @Valid
+    @GET
+    @Path("/meta/types/{type-id}/attributes")
+
+    @Produces({ "application/json" })
+    @ApiOperation(value = "Retrieve the meta attributes of a user store of a given user store type.", notes = "This API provides the capability to retrieve the attribute mappings of a given user store type.   <b>Permission required:</b>  *_/permission/admin ", response = UserStoreAttributeMapping.class, authorizations = {
+            @Authorization(value = "BasicAuth"),
+            @Authorization(value = "OAuth2", scopes = {
+
+            })
+    }, tags={ "Meta", })
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Successful Response.", response = UserStoreAttributeMapping.class),
```

Comment:
Is this the correct generated code for the YAML file? API response 400 is missing here but defined in the yaml file
https://github.com/wso2/identity-api-server/pull/296/files#diff-28025054edb6c52e9f93f6ec5a69e738c96e61eef86885dba8a75ebd6ab8719bR346

Reply:
fixed in https://github.com/wso2/identity-api-server/pull/307